RHIDP-6130: Update docs to refer to new secret name (#949)
redhat-developer/red-hat-developers-documentation-rhdh: modules/dynamic-plugins/proc-using-custom-npm-registry.adoc

You can install external plugins in an air-gapped environment by setting up a custom NPM registry.
====

You can configure the NPM registry URL and authentication information for dynamic plugin packages using a Helm chart. For dynamic plugin packages obtained through `npm pack`, you can use a `.npmrc` file.

> Using the Helm chart, add the `.npmrc` file to the NPM registry by creating a secret named `<release-name>-dynamic-plugins-npmrc` with the following content:

Reviewer: @coreydaley @jmagak Is <release-name> intended to be a variable? If so, let's follow the CCS / OCP doc guidelines for replaceable text (use italics, underscore, etc).
```suggestion
Using the Helm chart, add the `.npmrc` file to the NPM registry by creating a secret. For example:
```

Reviewer: Why are we telling the user what to name their plugin, especially if part of it is replaceable? Maybe we just tell them to create the secret and then provide an example of what the name could be?

[source,yaml,subs="+quotes,+attributes"]
----
apiVersion: v1
kind: Secret
metadata:

>   name: `<release-name>-dynamic-plugins-npmrc`

Reviewer: ```suggestion
  name: `_<release_name>_-dynamic-plugins-npmrc`
```
See guidelines for replaceable text: https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values
